wls-gui-wahllokalsystem/src/components/stimmabgabevermerke/TheUWBStimmabgabevermerkeErfassenTable.vue

```diff
@@ -0,0 +1,153 @@
+            :key="wahldaten.wahlID"
+          >
+            <v-text-field
+              disabled
+              label="Beurkundung"
+            />
+          </td>
+        </tr>
+        <tr
+          v-for="number in lowestNumberOfRowsOverAllWahldaten"
+          :key="number"
+        >
+          <td>Nr. {{ number + 1 }}</td>
+          <td
+            v-for="wahldaten in stimmabgabevermerke.wahldaten"
+            :key="wahldaten.waehlerverzeichnisNummer"
+          >
+            <v-number-input
+              v-model="wahldaten.vermerke[number - 1].stimmzettel[0].anzahl"
+            />
+          </td>
+        </tr>
+        <tr>
+          <td><b>Gesamt</b></td>
+          <td
+            v-for="totalCount in stimmabgabevermerkeTableTotalEachWahldaten"
+            :key="totalCount"
+          >
+            <b>{{ totalCount }}</b>
+          </td>
+        </tr>
+      </tbody>
+    </v-table>
+    <base-dialog
+      :visible="deleteDialog"
+      dialogtitle="Reduzierung der Blätteranzahl des Wählerverzeichnisses"
+      confirmtext="Trotzdem Löschen"
+      canceltext="Abbrechen"
+      icon="$information"
+      @cancel="deleteDialog = false"
+      @confirm="onDialogConfirmDeletingRows"
+      ><div>
+        Sie wollen Blätter löschen, für die Sie Stimmabgabevermerke eingetragen
+        haben. Wenn Sie diese löschen, werden dadurch auch die Werte für die
+        Stimmabgabevermerke gelöscht.
+      </div></base-dialog
+    >
+  </v-container>
+</template>
+
+<script setup lang="ts">
+import { storeToRefs } from "pinia";
+import { computed, onMounted, ref } from "vue";
+
+import BaseDialog from "@/components/common/dialogs/BaseDialog.vue";
+import { useRules } from "@/composables/common/rules.ts";
+import { useStimmabgabevermerkeStore } from "@/stores/stimmabgabevermerkeStore.ts";
+import { useWahlenStore } from "@/stores/wahlenStore.ts";
+
+const { minNumber, maxNumber, required } = useRules();
+
+const {
+  stimmabgabevermerke,
+  stimmabgabevermerkeTableTotalEachWahldaten,
+  lowestNumberOfRowsOverAllWahldaten,
+} = storeToRefs(useStimmabgabevermerkeStore());
+const { isAnyRowThatShouldBeDeletedFilled, changeRowCount } =
+  useStimmabgabevermerkeStore();
+const { getWahlNameOrBlankStringById } = useWahlenStore();
+
+onMounted(() => {
+  rowSize.value = lowestNumberOfRowsOverAllWahldaten.value + 1;
+});
+
+const deleteDialog = ref(false);
+const rowSize = ref<number | null>(null);
+
+const changeRowSizeButtonText = computed(() => {
+  if (
+    rowSize.value != null &&
+    rowSize.value - 1 > lowestNumberOfRowsOverAllWahldaten.value
+  ) {
+    return "Erhöhen";
+  } else if (
+    rowSize.value != null &&
+    rowSize.value - 1 < lowestNumberOfRowsOverAllWahldaten.value
+  ) {
+    return "Reduzieren";
+  } else {
+    return "Übernehmen";
+  }
+});
+
+const disableChangeRowSizeButton = computed(() => {
+  return rowSize.value == null || rowSize.value <= 0;
```

Comment:
aktuell bekommt man bei eingabe >250 zwar die error meldung kann aber trotzdem speichern
```suggestion
  return rowSize.value == null || rowSize.value <= 0 || rowSize.value > 250;
```